CodeEdit/Features/QuickOpen/Views/QuickOpenPreviewView.swift

```diff
@@ -28,6 +28,12 @@ struct QuickOpenPreviewView: View {
     }
 
     var body: some View {
-        CodeFileView(codeFile: document, isEditable: false)
+        if document.fileURL != nil {
+            if document.fileURL!.isImage() {
```

Comment:
Besides images, are there other types of files that can be previewed using `OtherFileView`?

Reply:
`OtherFileView` supports opening text files as well, but I thought `CodeFileView()` using `CodeEditTextView` would be better for text files rather than `OtherFileView`, a more complete list can be found [here](https://developer.apple.com/documentation/quicklook). exclude audio, video, iWork and MSOffice files since they don't work at the moment.